build.sh

```diff
@@ -9,12 +9,20 @@ echo ""
 echo "Installing NPM dependencies..."
 
 npm i
+npm run install
```

Comment:
That's strange, from what I assumed based on `napa` docs, it should run automatically when you run `npm install` or `npm i` so there should be no need to explicitly call `npm run install` :thinking: